src/styles/themes.ts

```
@@ -324,6 +324,26 @@ export const chakraTheme = ChakraTheme({
             bg: 'transparent.200',
           },
         },
+        ...['green', 'yellow', 'red'].reduce(
+          (acc, variant) => ({
+            ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)